src/WhiteLibrary/keywords/window.py

```diff
@@ -35,7 +48,5 @@ def close_window(self, window_title=None):
         ``window_title`` is the title of the window.
         """
         if window_title is not None:
-            self.state.window = self.state.app.GetWindow(window_title)
-
+            self.attach_window(window_title)
```

Comment:
This could be modified so that attached window would not change in case the window title is given.